lib/helpers.js

```diff
@@ -40,3 +41,13 @@ export const truncate = (string, length, append = '…') => {
   const limit = ~~length;
   return x.length > limit ? (x.slice(0, limit) + append) : x;
 };
+
+export const toQueryString = (options = {}) =>
+  stringify(options, {
+    arrayFormat: 'brackets',
+    sort: (a, b) =>
```

Comment:
Ah ha. Interesting pitfall from translating from an unsorted data format to a sorted one.